app/javascript/practice-select.vue

```diff
@@ -62,7 +62,10 @@ export default {
     },
     submit() {
       this.$nextTick(() => {
-        location.href = `${location.pathname}?all=true&solved=${this.solved}&practice_id=${this.selected.id}&title=${this.selected.title}`
+        const newUrl = new URL(location.href)
+        newUrl.searchParams.set('practice_id', this.selected.id)
+        newUrl.searchParams.set('title', this.selected.title)
+        location.href = newUrl
```

Comment:
https://github.com/fjordllc/bootcamp/pull/5049#issuecomment-1160531243 の件ですが、以下のようにしてみると、解決できるようなので共有させて頂きます🙏

```suggestion
        newUrl.searchParams.delete('page')
        location.href = newUrl
```

Reply:
コードの提案もありがとうございます🙏 参考にさせていただきました！

https://github.com/fjordllc/bootcamp/pull/5049/commits/d9d13611b4d9d2d160c0807d9fe82ba84dcbfdd0